runtime/src/tasks/qthreads/tasks-qthreads.c

```diff
@@ -183,6 +183,8 @@ static syncvar_t exit_ret = SYNCVAR_STATIC_EMPTY_INITIALIZER;
 
 void chpl_task_yield(void)
 {
+    extern void chpl_qsbr_checkpoint(void);
```

Comment:
It seems it would be preferable to #include the new qsbr header. Is that challenging for some reason?